.travis.yml

```diff
@@ -139,6 +154,11 @@ script:
         ./script/integration-tests minikube deployment
         ./script/integration-tests minikube kafka
         ;;
+      minikube_nats)
+        make nats-controller-image NATS_CONTROLLER_IMAGE=$NATS_CONTROLLER_IMAGE
+        ./script/integration-tests minikube deployment
+        ./script/integration-tests minikube nats
+        ;;
```

Comment:
the image should be done in the `GKE` scenario as well since it is the scenario that pushes the images and test everything

Reply:
also retag the image with the TRAVIS_TAG in the `after_success` hook, rename the file in the `before_deploy` and finally add the file in the `deploy.file` list

Reply:
Done.